NDB.Covid19/NDB.Covid19/Locales/lt.json

```diff
@@ -89,53 +89,55 @@
-  "LANGUAGE_SELECTION_SO": "Somali on Lithuanian",
-  "LANGUAGE_SELECTION_TI": "Tigrinya on Lithuanian",
-  "LANGUAGE_SELECTION_UR": "Urdu on Lithuanian",
-  "LANGUAGE_SELECTION_LIT": "Lithuanian on Lithuanian",
-  "DAILY_NUMBERS_HEADER": "lit1",
-  "DAILY_NUMBERS_TITLE_ONE": "lit2",
-  "DAILY_NUMBERS_SUBHEADER": "lit3",
-  "DAILY_NUMBERS_TITLE_TWO": "lit4",
-  "DAILY_NUMBERS_SUBTEXT_TWO": "lit5",
-  "DAILY_NUMBERS_TITLE_THREE": "lit6",
-  "DAILY_NUMBERS_SUBSUBHEADER": "lit7",
-  "KEY_FEATURE_ONE_UPDATE_NEW": "lit8",
-  "KEY_FEATURE_ONE_UPDATE_ALL": "lit9",
-  "KEY_FEATURE_ONE_LABEL": "lit10",
-  "KEY_FEATURE_TWO_UPDATE_NEW": "lit11",
-  "KEY_FEATURE_TWO_UPDATE_ALL": "lit12",
-  "KEY_FEATURE_TWO_LABEL": "lit13",
-  "KEY_FEATURE_THREE_UPDATE_NEW": "lit14",
-  "KEY_FEATURE_THREE_UPDATE_ALL": "lit15",
-  "KEY_FEATURE_THREE_LABEL": "lit16",
-  "KEY_FEATURE_FOUR_UPDATE_NEW": "lit17",
-  "KEY_FEATURE_FOUR_LABEL": "lit18",
-  "KEY_FEATURE_FIVE_LABEL": "lit19",
-  "KEY_FEATURE_FIVE_UPDATE_NEW": "lit20",
-  "KEY_FEATURE_FIVE_UPDATE_ALL": "lit21",
-  "KEY_FEATURE_SIX_LABEL": "lit22",
-  "KEY_FEATURE_SIX_UPDATE_ALL": "lit23",
-  "KEY_FEATURE_SEVEN_LABEL": "lit24",
-  "KEY_FEATURE_SEVEN_UPDATE_ALL": "lit25",
-  "KEY_FEATURE_NINE_LABEL": "lit28",
-  "KEY_FEATURE_NINE_UPDATE_NEW": "lit29",
-  "KEY_FEATURE_NINE_UPDATE_ALL": "lit30",
-  "KEY_FEATURE_TEN_LABEL": "lit31",
-  "KEY_FEATURE_TEN_UPDATE_NEW": "lit32",
-  "KEY_FEATURE_TEN_UPDATE_ALL": "lit33",
+  "LANGUAGE_SELECTION_NB": "Norvegų",
+  "LANGUAGE_SELECTION_NN": "Norvegų",
+  "LANGUAGE_SELECTION_EN": "Anglų",
+  "LANGUAGE_SELECTION_AR": "Arabų",
+  "LANGUAGE_SELECTION_PL": "Lenkų",
+  "LANGUAGE_SELECTION_SO": "Somaliečių",
+  "LANGUAGE_SELECTION_TI": "Tigrinų",
+  "LANGUAGE_SELECTION_UR": "Urdu",
+  "LANGUAGE_SELECTION_LT": "Lietuvių",
+  "DAILY_NUMBERS_HEADER": "Pagrindiniai skaičiai",
+  "DAILY_NUMBERS_TITLE_ONE": "COVID-19 statistika",
+  "DAILY_NUMBERS_SUBHEADER": "Patvirtintų atvejų, mirčių, testų ir priėmimų skaičius paskutinį kartą atnaujintas {0} {1} val.. Šaltinis: <a href=\"https://www.fhi.no/sv/smittsomme-sykdommer/corona/dags--og-ukerapporter/dags--og-ukerapporter-om-koronavirus/\">Norvegijos Visuomenės sveikatos institutas</a>.",
+  "DAILY_NUMBERS_TITLE_TWO": "Skiepijimas",
+  "DAILY_NUMBERS_SUBTEXT_TWO": "Paskutinį kartą skiepų skaičius atnaujintas {0} {1} val. Šaltinis: <a href=\"https://www.fhi.no/sv/smittsomme-sykdommer/corona/dags--og-ukerapporter/dags--og-ukerapporter-om-koronavirus/\">Norvegijos Visuomenės sveikatos institutas</a>.",
+  "DAILY_NUMBERS_TITLE_THREE": "„Smittestopp“",
+  "DAILY_NUMBERS_SUBSUBHEADER": "„Smittestopp“ skaičiai paskutinį kartą atnaujinti {0} {1} val. Apie skaičius skaitykite šiuo adresu<a href=\"https://smittestopp.no/dagenstall/\">www.smittestopp.no/dagenstall</a>",
+  "KEY_FEATURE_ONE_UPDATE_NEW": "{0}",
+  "KEY_FEATURE_ONE_UPDATE_ALL": "Iš viso: {0}",
+  "KEY_FEATURE_ONE_LABEL": "Patvirtinti atvejai",
+  "KEY_FEATURE_TWO_UPDATE_NEW": "not needed",
```

Comment:
can't we set it to empty string then?

Reply:
Looks like it's set to not needed for all languages, so it should probably be deleted from the sheet

Reply:
"not needed" rows is now deleted from the sheet

Reply:
updated